esrally/utils/process.py

```diff
@@ -60,13 +60,15 @@ def exit_status_as_bool(runnable, quiet=False):
         return False
 
 
-def run_subprocess_with_logging(command_line, header=None, level=logging.INFO, env=None):
+def run_subprocess_with_logging(command_line, header=None, level=logging.INFO, stdin=None, env=None):
```

Comment:
I didn't find any existing tests for this, so deferred -- for now -- writing new ones.